less/home-page.less

```diff
@@ -25,51 +37,94 @@
       border-radius: 20px;
       height: 100%;
     }
+
+    @media @phone {
+
+      .blast{
+        width: 100%;
+        height: 100%;
+      }
+
+      .sky{
+        display: none;
+      }
+
+      .launch{
+        display: none;
+      }
+
+    }
+
   }
 
-  .visit-planets {
-    padding: 15px;
+}
+
+.visit-planets {
+  padding: 15px;
+
+  h3 {
+    margin-top: 11px;
+  }
+
+  p {
+    margin-top: 13px;
+  }
+}
+
+.walk {
+  margin-top: 60px;
+  border-top: 1px solid @main-border-color;
+  padding: 50px 60px;
+  display: flex;
+  justify-content: center;
+
+  .walk-text {
+    width: 50%;
 
     h3 {
-      margin-top: 11px;
+      margin-bottom: 10px;
     }
 
     p {
-      margin-top: 13px;
+      margin-bottom: 26px;
     }
-  }
-
-  .walk {
-    margin-top: 60px;
-    border-top: 1px solid @main-border-color;
-    padding: 50px 60px;
-    display: flex;
-    justify-content: center;
 
-    .walk-text {
-      width: 50%;
+    @media @phone {
+      width: 100%;
+    }
+    .btn {
+      .button();
 
-      h3 {
-        margin-bottom: 10px;
+      @media @phone {
+        width: 100%;
+        height: 60px;
       }
 
-      p {
-        margin-bottom: 26px;
-      }
+    }
+  }
 
-      .btn {
-      }
+  .walk-img {
+    width: 46%;
+    margin-left: 4%;
+
+    @media @phone {
+      width: 100%;
+      order: -1;
+      margin: 0 0 3% 0;
     }
 
-    .walk-img {
-      width: 46%;
-      margin-left: 4%;
-      
-      img {
-        width: 100%;
-        height: auto;
-        border-radius: 10px;
-      }
+    img {
+      width: 100%;
+      height: auto;
+      border-radius: 10px;
     }
   }
-}// home
\ No newline at end of file
+
+  @media @phone{
+    flex-direction: column;
```

Comment:
I'm pretty sure the issue is the img tag inside the space-image div is not targeted to 100% and it is overflowing.  I have not tested it out yet though.